.github/workflows/ci.yml

```diff
@@ -71,6 +73,8 @@ jobs:
             ~/.cache/go-build
             ./.tools
           key: go-cache-${{ hashFiles('**/go.sum') }}
```

Comment:
The cache key here probably also needs to be updated similar to what is in line 44. Same for 131 🙂

Reply:
ah yeah, thanks! i thought i had them all, damn

Reply:
[fixed](https://github.com/grafana/grafana-ci-otel-collector/pull/231#pullrequestreview-2734952425). thanks for catching that. I clearly had a brain error and somehow started to think only `restore-keys` needed to be changed 🙃